### ISSUE TITLE
tronwatch 24 seed and long private key are invalid

### ISSUE THREAD

**Issue by europound** [TronLink/tronlink-extension#330]
please help!
after years I came back to check up on my tron trx
tronwatch is dead now
tronwatch 24 seed words are accepted by tronlink but address is different with 0 balance
tronwatch long private key is an invalid format tronlink says :(
tronscan shows that the old tronwatch address is ok and balance is ok
I am really stuck and need help
what to do?

**Comment by khaledxGo** [comment]
Hello

**Comment by europound** [comment]
where to get v tronwatch v1.0.13?
or maybe very old tronlink can handle 128bit key?

**Comment by sk4king** [comment]
Yes, please. I have an old private key from TronWatch Wallet and i need to import this wallet into TronLink

**Comment by alex500b** [comment]
old version 3. of tronlink for PC still works with old private keys
